crates/torii/graphql/src/object/inputs/where_input.rs

```diff
@@ -111,6 +111,6 @@ fn parse_integer(
 fn parse_string(input: ValueAccessor<'_>, type_name: &str) -> Result<FilterValue> {
     input
         .string()
-        .map(|i| FilterValue::String(i.to_string()))
+        .map(|i| FilterValue::String(format!("0x{:0>64}", i.strip_prefix("0x").unwrap_or(i))))
```

Comment:
Hmm i dont think this will correctly format integer values to hex. It will result in valid hex but since hex is base16, it will represent a different number. For example, if we pass `i == 10`, this will encode to `0x00...0010` but it should be `0x00..000A`

Reply:
Fixed on commit b06b8d5
Since `i` is a string value, First I have to parse it into `u128`, then format to `hex`
Note: I tried to parse into `u256`, but I got the error ` the trait LowerHex is not implemented for u256`